### DISCUSSION TITLE
Issued Documents: mappatura parametri tra v1 e v2

### DISCUSSION THREAD

**Discussion by mmonti-ff** [fattureincloud/api#228] [Q&A] [answered]
Salve,
stiamo migrando le chiamate a fatture in cloud di un nostro applicativo dalla versione v1 alla v2. 
Come risorsa usiamo solamente "Issued documents" (ex. Documenti emessi).
Tra i vari aspetti modificati tra le versioni 1 e 2 vediamo che sono anche stati rinominati e modificati i parametri da inserire nelle chiamate agli endpoint. 
Per chiarezza a titolo di esempio:
-  nome > entity.name
- indirizzo_via > entity.address_street
- lista_articoli{} > items_list{}

Volevamo capire se avete a disposizione una mappatura di questi campi in modo da capire in modo certo come sono stati trasformati i campi.
Se non fosse disponibile una documentazione c'è un modo per ricostruire queste modifiche? 
Cercando domande similari ad esempio vedo che [qui](https://github.com/fattureincloud/api/discussions/217) è stato consigliato di creare il documento con la V1 e poi fare la get con V2 se non erro.

Grazie,
Matteo

**Comment by valmoz** [comment, marked as answer]
Buongiorno @mmonti-ff 
esatto, non abbiamo a disposizione una mappatura completa di tutti campi tra le V1 e le V2.

Quello che possiamo suggerirvi è di iniziare seguendo quanto suggerito in [questa guida](https://developers.fattureincloud.it/docs/api-v2-migration), passando poi ad effettuare quanto suggerito dal mio collega nell'altra discussion (creazione con le V1, lettura con le V2) per fugare ogni dubbio.

La comparazione tra il risultato offerto dalle V2 e la richiesta effettuata con le V1 vi permetterà di identificare il campo da utilizzare nell'effettuazione della richiesta V2.

Grazie
Mauro
